lib/client-assets.php

```diff
@@ -745,7 +745,8 @@ function gutenberg_extend_block_editor_styles_html() {
 
 	ob_start();
 
-	wp_styles()->done = array();
+	// We do not need reset styles for the iframed editor.
+	wp_styles()->done = array( 'wp-reset-editor-styles' );
```

Comment:
This is the easiest way to get the job done. We trick `do_items` in thinking that the handle has already been added, so it will be skipped.